### ISSUE TITLE
sass-loader bump version in pull request #453 is causing build failure.

### ISSUE THREAD

**Issue by maxerbox** [nuxt-community/vuetify-module#484]
**Module version**
1.12.2

**Describe the bug**
sass-loader bump version in pull request #453 is causing build failure.

```
in ./node_modules/vuetify/src/components/VApp/VApp.sass

Module build failed (from ./node_modules/@nuxtjs/vuetify/node_modules/sass-loader/dist/cjs.js):
TypeError: this.getOptions is not a function
    at Object.loader (/mnt/6c613b3f-1cec-40c0-a5fb-0e771e22f7c2/home/mxb/Documents/xxx/packages/wb-ui/node_modules/@nuxtjs/vuetify/node_modules/sass-loader/dist/index.js:25:24)

 @ ./node_modules/vuetify/src/components/VApp/VApp.sass 4:14-232 15:3-20:5 16:22-240
 @ ./node_modules/vuetify/lib/components/VApp/VApp.js
 @ ./node_modules/vuetify/lib/components/VApp/index.js
 @ ./layouts/error.vue
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi ./node_modules/eventsource-polyfill/dist/browserify-eventsource.js (webpack)-hot-middleware/client.js?reload=true&timeout=30000&ansiColors=&overlayStyles=&path=%2F__webpack_hmr%2Fclient&name=client ./.nuxt/client.js
```

**Additional context**
sass-loader from version 11 is only compatible with webpack 5.

I'm using nuxt 2

It should be considered as a breaking change

**Comment by mschoeffmann** [comment]
same here. reverted to 1.12.1

**Comment by dgrabla** [comment]
Yeah, same here. A temporal workaround that works for us is to install an older version of the dependency that causes the problem as a direct dependency of your project: 
```
> npm install --save sass-loader@10.2.0
```

**Comment by n1ngu** [comment]
- @nuxtjs/vuetify@1.12.2 requires sass-loader@12.3.0 ( https://github.com/nuxt-community/vuetify-module/pull/453 )
- that only supports webpack@5 or higher ( https://github.com/webpack-contrib/sass-loader/issues/974#issuecomment-874578558 )
- but nuxt@2 has an overall restriction on webpack@4 ( https://github.com/nuxt/nuxt.js/issues/9284 )

Am I missing something or this should be reverted?

**Comment by rahulkumarsingh73690** [comment]
+1

**Comment by n1ngu** [comment]
An immediate action could be yanking @nuxtjs/vuetify@1.12.2 from npm so that no more people runs into this bug. And also to prevent any dependency updating workflow to resolve this exact version while saving the whole user community from the burden of either freezing to 1.12.1 or requiring sass-loader@10 and committing to undo any workaround in the near future.

@kevinmarrec sorry to bother you. Which maintainer should be pinged for this task?

**Comment by clarkdo** [comment]
v1.2.3 has been released with reverting sass-loader to v10

**Comment by maxerbox** [comment]
closed by #487
